src/main/java/com/hazelcast/web/WebFilterConfig.java

```diff
@@ -37,30 +37,135 @@
+    public static final String SESSION_TTL_SECONDS = "session-ttl-seconds";
+
+    /**
+     * <p>Configuration key: <code>use-client</code></p>
+     * Specifies whether you want to connect to an existing cluster as a client.
+     * Its default value is false.
+     */
+    public static final String USE_CLIENT = "use-client";
+
+    /**
+     * <p>Configuration key: <code>instance-name</code></p>
+     * Name of an existing Hazelcast instance, if you want to use it.
+     * Its default value is null.
+     * If you do not have an instance, then you should create one.
+     */
+    public static final String INSTANCE_NAME = "instance-name";
+
+    /**
+     * <p>Configuration key: <code>map-name</code></p>
+     * Name of the distributed map storing your web session objects.
+     */
+    public static final String MAP_NAME = "map-name";
+
+    /**
+     * <p>Configuration key: <code>sticky-session</code></p>
+     * If set to true, all requests of a session are routed to the member where the session is first created.
+     * If set to false, when a session is updated on a member, entry for this session on all members is invalidated.
+     * You have to know how your load balancer is configured before setting this parameter.
+     * Its default value is true.
+     */
+    public static final String STICKY_SESSION = "sticky-session";
+
+    /**
+     * <p>Configuration key: <code>config-location</code></p>
+     * Location of Hazelcast configuration. It can be specified as a servlet resource, classpath resource or as a URL.
+     * Its default value is hazelcast-default.xml or hazelcast.xml in the classpath.
+     */
+    public static final String CONFIG_LOCATION = "config-location";
+
+    /**
+     * <p>Configuration key: <code>shutdown-on-destroy</code></p>
+     * Specifies whether you want to shut down the Hazelcast instance during the undeployment of your web application.
+     * Its default value is true.
+     */
+    public static final String SHUTDOWN_ON_DESTROY = "shutdown-on-destroy";
+
+    /**
+     * <p>Configuration key: <code>deferred-write</code></p>
+     * Specifies whether the sessions in each instance will be cached locally. Its default value is false.
+     */
+    public static final String DEFERRED_WRITE = "deferred-write";
+
+    /**
+     * <p>Configuration key: <code>use-request-parameter</code></p>
+     * Specifies whether a request parameter can be used by the client to send back the session ID value.
+     * Its default value is false.
+     */
+    public static final String USE_REQUEST_PARAMETER = "use-request-parameter";
+
+    /**
+     * <p>Configuration key: <code>transient-attributes</code></p>
+     * If you have some attributes that you do not want to be distributed, you can mark those attributes as transient.
+     * Transient attributes are kept in the server and when the server is shutdown, you lose the attribute values.
+     * Values can be defined comma separated. Its default value in null
```

Comment:
Let's make this a bit formal
```suggestion
     * Comma separated attributes not to be written to distributed map
     * but only to be kept locally in a server, not visible to other servers.
     * The default value is an empty list.
```